### ISSUE TITLE
Is my parameter setting correct？Why is my ckpt file larger than yours？

### ISSUE THREAD

**Issue by pipeser** [john852517791/pytorch_lightning_FAD#7]
| Name           | Type             | Params | Mode 
------------------------------------------------------------
0 | model          | Model            | 341 M  | train
1 | loss_criterion | CrossEntropyLoss | 0      | train
------------------------------------------------------------
25.9 M    Trainable params
315 M     Non-trainable params
341 M     Total params
1,365.436 Total estimated model params size (MB)
533       Modules in train mode
406       Modules in eval mode

Why is my ckpt file larger than yours？What did I do wrong
hparams.yaml：
N_f: 5
P: 10
SNRmax: 40
SNRmin: 10
algo: 3
batch_size: 4
da_prob: 2
data_module: utils.loadData.asvspoof_data_DA
epochs: 50
eta: 0.0
g_sd: 2
gamma: 0.1
gpuid: '0'
inference: false
loss: WCE
loss_lr: 0.01
loss_weight: 0.0
maxBW: 1000
maxBiasLinNonLin: 20
maxCoeff: 100
maxF: 8000
maxG: 0
minBW: 100
minBiasLinNonLin: 5
minCoeff: 10
minF: 20
minG: 0
module_model: models.w2v2_moe_fz24_aasist
moe_exp_hid: 128
moe_experts: 4
moe_topk: 2
momentum: 0.9
nBands: 5
no_best_epochs: 3
num_warmup_steps: 3
optim: adamw
optim_lr: 1.0e-05
reduce: 0
rho: 0.5
savedir: a_l/noft_24/2_4_128/
scheduler: cosWarmup
seed: 888
step_size: 5
testset: LA21
tl_model: models.tl_model_moe
total_step: 1057
trained_model: ./a_l/noft_24/2_4_128/version_3
truncate: 64600
usingDA: true
weight_decay: 0.0001

**Comment by john852517791** [comment]
I manually removed the Wav2vec 2.0 weights from the checkpoint, as that part was frozen during training.

**Comment by john852517791** [comment]
btw, the title of the issue is misleading, maybe "ckpt file size problem" will be better

**Comment by pipeser** [comment]
> I manually removed the Wav2vec 2.0 weights from the checkpoint, as that part was frozen during training.
Thank you for your reply. I have verified it once。19la=1.3,  21la=4.7,  21df=3.4 
maybe i need train n more times

**Comment by john852517791** [comment]
good luck
